client/src/components/avatar/avatar.jsx

```diff
@@ -0,0 +1,13 @@
+import s from './avatar.module.sass';
+
+const Avatar = ({url}) =>{
+    return(
+        <>
```

Comment:
why this need to be wrapped in Fragment?

Reply:
I corrected